pkg/schema/private/ekscluster_kfd_v1alpha2.go

```diff
@@ -743,6 +743,9 @@ type SpecInfrastructureVpcNetworkSubnetsCidrs struct {
 }
 
 type SpecInfrastructureVpn struct {
+	// BucketNamePrefix corresponds to the JSON schema field "bucketNamePrefix".
+	BucketNamePrefix *TypesAwsS3BucketName `json:"bucketNamePrefix,omitempty" yaml:"bucketNamePrefix,omitempty"`
+
 	// DhParamsBits corresponds to the JSON schema field "dhParamsBits".
```

Comment:
You should create a new API version because the v1alpha2 has already been released.